foundation/foundation-upgrade/src/test/resources/test-files/v1_9_0/ArgocdValueFileSyncPolicyMigration/migration/with-automated-prune-values.yaml

```
@@ -0,0 +1,17 @@
+spec:
+  destination:
+    server: https://kubernetes.default.svc
+  targetRevision: dev
+  repo: test.url
+  helm:
+    valueFiles:
+      - values.yaml
+  syncPolicy:
+    syncOptions:
+      - CreateNamespace=true
+      - ApplyOutOfSyncOnly=true
+    automated:
+      prune
```

A: Because this is a yml list, the proper format would be to precede with `-`. Doesn't really matter much as we're just confirming that the content is preserved.
```suggestion
      - prune
```

---

ah this should be `prune: true`. Let me correct it